### DISCUSSION TITLE
Use Chezmoi in a team

### DISCUSSION THREAD

**Discussion by guettli** [twpayne/chezmoi#3460] [Q&A]
I guess the major use-case for `chezmoi` is the single developer, setting up the environment for himself.

We want to use chezmoi in a team.

Now we face this:

How can one person have a slightly different dot-file?

Is there a way to mark some line like this:

```
... common config

### start ignore chezmoi
...
my personal config (not part of the git repo)
...
### end ignore chezmoi

... common config
```

Maybe I am blind, but I have not found something like this in the docs.

**Comment by twpayne** [comment]
chezmoi is a dotfile manager for individuals. Although some people are using it for teams, this is not what chezmoi is designed for.

For your particular case, the way to handle files that vary from machine to machine (e.g. between users) is to use templates. In this particular case, you can use the `include` template function, something like:

```
{{ $personalConfigFile := printf "%s/%s.txt" .chezmoi.homeDir .chezmoi.username -}}
{{ if stat $personalConfigFile -}}
{{   include $personalConfigFile -}}
{{ end -}}
```

**Comment by halostatue** [comment]
The main person active in these discussions is @lockszmith-gh, who has written a short guide on their approach: https://github.com/twpayne/chezmoi/discussions/2574

**Reply by Lockszmith-GH** [reply]
Thanks for the shout-out @halostatue.

Indeed, it has worked well in the team I built this for.

Ask away on the #2574 discussion
